charts/grafana/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 6.47.0
+version: 6.47.1
```

New feature should bump to minor version.

```suggestion
version: 6.48.0
```

---

@zanhsieh It seems that when you merged main into this branch you already bumped to version 6.48.0.

Should I bump to 6.49.0 or can I resolve this?

---

6.48.0 should be fine.